src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -470,7 +470,7 @@ private async Task<(long, DatabaseTopology, List<string>)> CreateDatabase(string
                 {
                     topology = new DatabaseTopology
                     {
-                        Members = nodeUrlsAddedTo,
+                        Members = members
```

Comment:
This is a change in behavior, no?

Reply:
no, we always put array of node tags in members property of topology, it was fixed in the master branch as well
https://github.com/ravendb/ravendb/blob/9c784dbc2ba758bce506c3099fa45061f232d3a4/src/Raven.Server/Web/System/AdminDatabasesHandler.cs#L422